lib/matplotlib/dates.py

```diff
@@ -1338,7 +1338,11 @@ def get_locator(self, dmin, dmax):
             self._freq = freq
 
             if self._byranges[i] and self.interval_multiples:
-                byranges[i] = self._byranges[i][::interval]
+                if i == 2 and interval == 14:
+                    # special case for monthday: Just tick 1 and 15:
+                    byranges[i] = range(1, 16, 14)
```

Comment:
Could this not simply be [1, 15] to make it more clear?

Reply:
Yep!